viewer/packages/api/src/public/migration/Cognite3DViewer.ts

```diff
@@ -1198,39 +1198,35 @@ export class Cognite3DViewer {
     // TODO christjt 03-05-2021: This seems ridiculous, and the number seems to be pulled out of thin air.
     // On low end it might not downscale enough, and on high end it looks bad / blurred.
     // For the love of God someone move this to the render manager and make it dynamic based on the device.
-    const maxTextureSize = 1.4e6;
+    const maxPhysicalFrameBufferSize = 1.4e6;
 
-    const rendererSize = this.renderer.getSize(new THREE.Vector2());
-    const devicePixelRatio = this.renderer.getPixelRatio();
-    const rendererPixelWidth = rendererSize.width;
-    const rendererPixelHeight = rendererSize.height;
+    const virtualFramebufferSize = this.renderer.getSize(new THREE.Vector2());
+    const pixelRatio = window.devicePixelRatio;
```

Comment:
Should it always be window, or should it be possible to set using a custom `rendered`?